metadata-db-app/src/migrations/068_add_fingerprint_to_variables_table.clj

```diff
@@ -0,0 +1,17 @@
+(ns migrations.068-add-fingerprint-to-variables-table
+  "Adds fingerprint to variables table"
+  (:require
+   [config.mdb-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up to version 68."
+  []
+  (println "migrations.068-add-fingerprint-to-variables-table up...")
+  (h/sql "alter table cmr_variables add fingerprint VARCHAR(64)")
+  (h/sql "CREATE INDEX variables_FPI ON cmr_variables(fingerprint)"))
```

Comment:
I can't tell if the unique-field constraint provides the provider in the query. Can you add some debug logging and run a test against external Oracle?

The SQL seems to be generated in the end by `gen-find-concepts-in-table-sql` in `cmr.metadata-db.data.oracle.search`.

I just want to make sure the index we add matches that query.

Reply:
Yes, you are right. The query does include provider_id. Thanks!